app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -36,13 +60,22 @@ define([
      */
     checkPasswordStrength: function (password) {
       var self = this;
+      if (! this._isPasswordStrengthCheckEnabled()) {
+        return p('DISABLED');
+      }
+
       return this._getPasswordStrengthChecker()
         .then(function (passwordStrengthChecker) {
           var deferred = p.defer();
           passwordStrengthChecker(password, function (passwordCheckStatus) {
             // in the future, do some fancy tooltip here.
-            self.logScreenEvent('experiment.pw-strength.' + passwordCheckStatus);
-            console.log('experiment.pw-strength.' + passwordCheckStatus);
+            passwordCheckStatus = passwordCheckStatus || 'UNKNOWN';
+
+            var eventName = 'experiment.pw_strength.' +
```

Comment:
Nope, since we use `logScreenEvent` to log the event, the screen name is prepended automatically.